src/action.js

```diff
@@ -100,14 +107,16 @@ function markdownReport(reports, commit, options) {
         showMissingMaxLength > 0
           ? crop(file.missing, showMissingMaxLength)
           : file.missing;
-      files.push([
-        escapeMarkdown(showClassNames ? file.name : file.filename),
-        `\`${fileTotal}%\``,
-        showLine ? `\`${fileLines}%\`` : undefined,
-        showBranch ? `\`${fileBranch}%\`` : undefined,
-        status(fileTotal),
-        showMissing ? (fileMissing ? `\`${fileMissing}\`` : " ") : undefined,
-      ]);
+      if (false === onlySummary) {
```

Comment:
We can add this check to the `report.files.filter` instead, like `!onlySummary &&..`